packages/@vue/cli/lib/util/mergeDeps.js

```diff
@@ -45,6 +52,8 @@ module.exports = function resolveDeps (generatorId, to, from, sources) {
 
 const leadRE = /^(~|\^|>=?)/
 const rangeToVersion = r => r.replace(leadRE, '').replace(/x/g, '0')
+const extractSemver = r => r.replace(/^.+#semver:/, '')
```

Comment:
shouldn't `semver:` be optional here?

Reply:
according to the npm docs (which do not seem to be 100% accurate),  if it is just `#...` and not `#semver:...` then it is referring to a commit hash, not a semver